sdk/src/akkaserverless.jsdoc

```diff
@@ -99,3 +99,28 @@
  *                             distributed to remaining replicas in the background.
  * @property {number} ALL Updates will be written immediately to all replicas.
  */
+
+/**
```

Comment:
Needs to be 1:1 duplicated i the jsdoc or else it is no available from the compiled/generated TS API.